### PR TITLE
Adding test for increasing the coverage for speechtotext component

### LINKED ISSUE #1467: Add test for increasing coverage
**I'm submitting a ...**
- [x] bug report
- [x] feature request

**Current behavior:**

currently there are some services and components whost test coverage is very low so we have write test for increasing the test coverage

**Will work on the issue:**
yes

### PR DESCRIPTION
**Pull request by subhahu123**
Fixes #1467

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- https://pr-1458-fossasia-susper.surge.sh

### CODE REVIEW

File: src/app/speechtotext/speechtotext.component.spec.ts [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+import { Router } from '@angular/router';
+import * as queryactions from '../actions/query';
+import * as speechactions from '../actions/speech';
 
 describe('SpeechtotextComponent', () => {
   let component: SpeechtotextComponent;
   let fixture: ComponentFixture<SpeechtotextComponent>;
+  let message: any;
+  let ticks: any;
+  let timer: any;
```
Comment (simsausaurabh) [inline src/app/speechtotext/speechtotext.component.spec.ts:17]: Don't just define type as ```any```, always try to declare exact type of variable. And please also fix it at all instances.

File: src/app/speechtotext/speechtotext.component.spec.ts [hunk 2]
```diff
@@ -31,4 +47,89 @@ describe('SpeechtotextComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should create div with id spch', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('div#spch')).toBeTruthy();
```
Comment (simsausaurabh) [inline src/app/speechtotext/speechtotext.component.spec.ts]: You should write tests which tests something meaningful.

**Review by simsausaurabh — CHANGES_REQUESTED** [review]
You also need to squash your commits, and fill the PR description properly.

File: src/app/speechtotext/speechtotext.component.spec.ts (hunk 2 above)
Comment (subhahu123) [reply src/app/speechtotext/speechtotext.component.spec.ts]: removed

File: src/app/speechtotext/speechtotext.component.spec.ts (hunk 1 above)
Comment (subhahu123) [reply src/app/speechtotext/speechtotext.component.spec.ts:17]: any is used in component.ts file i m just keeping it sync with it
`

**Review by praveenojha33 — APPROVED** [review]
LGTM :+1:

**Review by sakshee-19 — APPROVED** [review]

**Review by simsausaurabh — APPROVED** [review]

**Pull request merged**